src/components/Analysis/AnalysisListItem.jsx

```diff
@@ -48,7 +48,7 @@ const AnalysisListItem = ({ sortedTimeserie, totalCO2, type }) => {
   const travelCount = sortedTimeserieValue.timeseries.length
   const CO2 = sortedTimeserieValue.totalCO2
 
-  const CO2percent = `${Math.round((CO2 * 100) / totalCO2)}%`
+  const CO2percent = CO2 > 0 ? `${((CO2 * 100) / totalCO2).toFixed(2)}%` : `0%`
```

Comment:
```suggestion
  const CO2percent = totalCO2 > 0 ? `${((CO2 * 100) / totalCO2).toFixed(2)}%` : `0%`
```
I think C02 = 0  is not a problem.  But if totalCO2 = 0, it crashes.

Reply:
nit: we could add 2 little tests on this feature:  
- one with totalCO2 = 0, to check that there is no crash 
- one with CO2percent = 0.06

Reply:
Fix with new implementation 👍